Later version for 4.8.1 grabbed from Debian:
https://sources.debian.org/patches/netcdf/1:4.8.1-1/byteswap.patch/

Earlier version used:
https://github.com/DennisHeimbigner/netcdf-c/commit/6f86660da86f279cad1f2804cf3242767de27630.patch
https://github.com/Unidata/netcdf-c/issues/1687
https://bugs.gentoo.org/772863

Description: Add forward declarations for byteswap8 & byteswap4.
Author: Dennis Heimbigner
Bug: https://github.com/Unidata/netcdf-c/issues/1687

--- a/libdispatch/dfilter.c
+++ b/libdispatch/dfilter.c
@@ -20,6 +20,9 @@
 #include "hdf5internal.h"
 #endif
 
+static void byteswap8(unsigned char*);
+static void byteswap4(unsigned char*);
+
 /*
 Unified filter related code
 */
